pynucastro/networks/tests/conftest.py

```diff
@@ -34,12 +34,12 @@ def compare_network_files(request):
     update_networks = request.config.getoption("--update-networks")
 
     def _compare_network_files(test_path, ref_path, skip_files=()):
-        base_path = os.path.relpath(os.path.dirname(__file__))
-        ref_path = os.path.join(base_path, ref_path)
+        base_path = Path(__file__).relative_to(Path.cwd())
+        ref_path = base_path/ref_path
 
         skip_files = set(skip_files)
-        test_files = set(os.listdir(test_path)) - skip_files
-        ref_files = set(os.listdir(ref_path)) - skip_files
+        test_files = set(test_path.iterdir()) - skip_files
+        ref_files = set(ref_path.iterdir()) - skip_files
```

Comment:
These should be file names, not full paths:
```suggestion
        test_files = set(os.listdir(test_path)) - skip_files
        ref_files = set(os.listdir(ref_path)) - skip_files
```